cvat/apps/engine/backup.py

```diff
@@ -967,6 +969,11 @@ def export(db_instance, request, queue_name):
             # in case the server is configured with ONE_RUNNING_JOB_IN_QUEUE_PER_USER
             # we have to enqueue dependent jobs after canceling one
             rq_job.cancel(enqueue_dependents=settings.ONE_RUNNING_JOB_IN_QUEUE_PER_USER)
+            try:
+                send_stop_job_command(rq_job.connection, rq_job.id)
+            except InvalidJobOperation:
+                pass
```

Comment:
May be replaced with [suppress](https://docs.python.org/3/library/contextlib.html#contextlib.suppress) from contextlib
```suggestion
            with suppress(InvalidJobOperation):
                send_stop_job_command(rq_job.connection, rq_job.id)
```